Maquetacion/scss/footer.scss

```diff
@@ -0,0 +1,29 @@
+.footer-bar {
+  align-items: center;
+  background: $blue;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  justify-content: center;
+  margin-top: 30px;
+  padding-right: 20px;
```

Comment:
Try to use simmetric values whenever it is possible in order to handle properly LTR and RTL writing systems. In this case, try using the shorthand for horizzontal and vertical paddings. (Same goes for the margin).

Reply:
👍